src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -87,7 +89,7 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
         if (this.participationUpdateListener) {
             this.participationUpdateListener.unsubscribe();
             if (this.studentParticipation) {
-                this.participationWebsocketService.unsubscribeForLatestResultOfParticipation(this.studentParticipation.id);
+                this.participationWebsocketService.unsubscribeForLatestResultOfParticipation(this.studentParticipation.id, this.exercise!!);
```

Comment:
What do two `!!` mean? Is this meant to be a force unwrap, only needing one `!`?
```suggestion
                this.participationWebsocketService.unsubscribeForLatestResultOfParticipation(this.studentParticipation.id, this.exercise!);
```

Reply:
Yes, you are right. Seem I was still in Kotlin mode, where you need `!!` :D